.github/actions/prepare-frontend/action.yml

```diff
@@ -45,3 +52,6 @@ runs:
       if: inputs.apply-patches == 'true'
       run: yarn patch-package
       shell: bash
+    - name: Generate OpenAPI TypeScript types
```

Comment:
what is this for ? @chodorowicz

Reply:
@qnkhuat we're not committing generated TS file with all the types to the repo, that's why we need to create it at the pre-build step.